src/screens/OnboardingPages/OnboardingActivitiesPage.js

```diff
@@ -0,0 +1,45 @@
+import { Grid, Typography } from "@mui/material";
+import Button from "@mui/material/Button";
+import activitiesPage from './../../assets/onboardingImages/activitiesPage.svg';
+
+const OnboardingActivitiesPage = () => {
+  return (
+    <Grid
+      container
+      direction="column"
+      justifyContent="center"
+      alignItems="center"
+      spacing={2}
+      p={8}
+    >
+      <Grid item xs={12}>
+        <Typography variant="h1">Practice skills with activities</Typography>
+      </Grid>
+
+      <Grid item xs={12}>
+        <img src={activitiesPage} alt="activities page" height={650} />
+      </Grid>
+
+      <Grid item xs={12}>
+        <Typography variant="body1">
+          Each SEL pillar has its own page with related activities for your child to practice that specific skill. You can learn about how the activity builds the skill, print out the instructions, watch an instructional video, and track your child&apos;s completion!
+        </Typography>
+      </Grid>
+
+      <Grid item xs={12}>
+      <Link to="../HomePage.js">
+          <Button variant="text">Skip tutorial</Button>
+        </Link>
+      </Grid>
```

Comment:
This isn't how links should reference internal links. Should be in the format of the link itself not the component file!